ads/google/a4a/traffic-experiments.js

```diff
@@ -121,7 +121,16 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
  */
 function maybeSetExperimentFromUrl(win, experimentName,
     controlBranchId, treatmentBranchId, manualId) {
-  const expParam = parseQueryString(win.location.search)['exp'];
+  let queryString = win.location.search;
+  // If we're inside the AMP viewer, the hash fragment will have been moved
+  // to the originalHash field.
+  const hash = win.location.originalHash || win.location.hash;
```

Comment:
I don't disagree.  Is this something that's near enough term that I should block this on?  (Would rather not, since this PR is required for the full A4A experiment to get running.)